Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAutomationWorker.py

```diff
@@ -36,7 +37,7 @@ def set_marshall_helper(WorkspaceId, Enabled, AzureDnsAgentSvcZone, mock_worker_
     try:
         is_primary = is_oms_primary_workspace(WorkspaceId)
     except Exception, e:
-        log(ERROR, "Could not detect if workspace is primary\n %s" %(str(e)))
+        log(ERROR, "Could not detect if workspace is primary\n %s" % (str(e)))
```

Comment:
Actually, line 223 does do it like that: https://github.com/Microsoft/PowerShell-DSC-for-Linux/blob/15e74d3e344848b57b36158d9ad7bc8eb20162dc/Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAutomationWorker.py#L223